homeassistant/components/amazon_polly/const.py

```diff
@@ -1,14 +1,12 @@
 """Constants for the Amazon Polly text to speech service."""
-from __future__ import annotations
+import typing
```

Comment:
This is an unrelated style change, that has an effect on this whole PR (making this PR unneeded hard to review as well). We generally import specifics for typing, instead of the module.

Please revert these styling changes from this PR.